Source/WebCore/rendering/style/RenderStyle.cpp

```diff
@@ -3465,6 +3465,20 @@ const CSSCustomPropertyValue* RenderStyle::customPropertyValue(const AtomString&
     return nullptr;
 }
 
+bool RenderStyle::customPropertyValueEqual(const RenderStyle& other, const AtomString& name) const
+{
+    if (&nonInheritedCustomProperties() == &other.nonInheritedCustomProperties() && &inheritedCustomProperties() == &other.inheritedCustomProperties())
```

Comment:
Is it different from `RenderStyle::customPropertiesEqual()` ?

Reply:
`customPropertiesEqual` tests for true equality (which may involve checking every single property). This just does the fast pointer equality test as a shortcut (custom property data is often shared).